src/test/java/com/vaadinÐguice/server/UIScopeTest.java

```diff
@@ -1,193 +0,0 @@
-package com.vaadin.guice.server;
```

Comment:
It looks like commit 6edd44b added a bogus folder under src/test/java/com. If you have a look at what's on github, there are 2 folders: vaadin/guice and vaadin�guice. The second one seems to be a mistake. That's what I removed.